src/Assignment-4/assignment4_test.js

```diff
@@ -64,16 +64,9 @@
     it("container should have header for page", function () {
       assert.isTrue(isExist(header_ele));
     });
-    it("container should have logo image for page", function () {
+    it("header navigation should have three list items", function () {
       assert.isTrue(
-        isExist(
-          frame.toDomElement().contentDocument.getElementsByTagName("img")
-        )
-      );
-    });
-    it("input should have name attribute", function () {
-      assert.isTrue(
-        frameDom.getElementsByTagName("input")[0].hasAttribute("name")
+          frame.toDomElement().contentDocument.querySelectorAll("header nav ul")[0].childElementCount=== 3
```

Comment:
Assignment-4 has already been given to students and it is completed. Why are we modifying these test cases?

Reply:
Some of the students faced generic issues related to tag counts and classes, so we made them more precise.